pkg/backplaneapi/clientUtils.go

```diff
@@ -100,14 +85,7 @@ func (s *DefaultClientUtilsImpl) MakeRawBackplaneAPIClient(base string) (Backpla
 }
 
 func (*DefaultClientUtilsImpl) MakeBackplaneAPIClientWithAccessToken(base, accessToken string) (BackplaneApi.ClientWithResponsesInterface, error) {
-	co := func(client *BackplaneApi.Client) error {
-		client.RequestEditors = append(client.RequestEditors, func(ctx context.Context, req *http.Request) error {
-			req.Header.Add("Authorization", "Bearer "+accessToken)
-			req.Header.Set("User-Agent", "backplane-cli"+info.Version)
-			return nil
-		})
-		return nil
-	}
+	co := makeClientOptions(accessToken)
```

Comment:
I guess it's not good to mix constructor method with non-constructor methods; what if call this as `s.makeClientOptions()`